test/api/dependency/api_test.go

```diff
@@ -0,0 +1,96 @@
+package dependency
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestDependencyCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Dependency_CRUD", func(t *testing.T) {
```

Comment:
Just a nit, but both Dependency and CRUD is part of the method name (2 lines above), so it is already part of test output, so it would be better add a new information instead. I'd suggest update to something like ApplicationFrom.Name->ApplicationTo.Name (with fmt.Sprintf).